packages/host/src/Instance.ts

```diff
@@ -787,16 +817,17 @@ rcon.print(game.table_to_json(players))`.replace(/\r?\n/g, " ");
 	async prepare() {
 		this.logger.verbose("Writing server-settings.json");
 		await this.writeServerSettings(true);
+		this._expectedUserUpdates = [];
 
-		if (this.config.get("factorio.sync_adminlist")) {
+		if (this.config.get("factorio.sync_adminlist") !== "Disabled") {
```

Comment:
Rather than use a consttant dict, I have updated the type on the interface to be a union of strings. This will cause typescript to error if you attempt to test for a string not in the union.